tests/granulate_utils/test_cgroups.py

```diff
@@ -0,0 +1,39 @@
+from typing import Callable, Dict
+from unittest.mock import patch
+
+from granulate_utils.linux.cgroups.cpu_cgroup import CpuCgroup
+
+
+def wrap_read_from_control_file(mapping: Dict[str, str]) -> Callable:
+    def func(tunable_name: str) -> str:
+        return mapping[tunable_name]
+
+    return func
+
+
+@patch("granulate_utils.linux.cgroups.cpu_cgroup.BaseCgroup.__init__", return_value=None)
```

Comment:
Maybe patch only `_verify_preconditions`? (I figure that's the reason behind this patch)